frontend/src/Components/ResultsPage/ResultsPage.module.css

```diff
@@ -15,3 +15,8 @@
   color: var(--text-primary);
   fill: var(--text-primary);
 }
+
+.pendingConnection {
+  color: var(--text-tertiary);
```

Comment:
In dark mode, I find that the contrast between connected and pending users is very subtle. I wonder if we want to improve here e.g. by making `--text-tertiary` a little darker only in dark mode.

Reply:
Good suggestion. Please take another look.